entwatch/ze_roof_adventure_v8f.cfg

```diff
@@ -0,0 +1,314 @@
+		"forcedrop"		"true"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"514832"
+		"mode"			"3"
+		"maxuses"		"1"
+		"cooldown"		"0"
+		"maxamount"		"1"
+	}
+	"9"
+	{
+		"name"			"Unlimited Ammo"
+		"shortname"		"Ammo"
+		"color"			"{lime}"
+		"buttonclass"		""
+		"filtername"		"ammo_user"
+		"hasfiltername"		"true"
+		"blockpickup"		"false"
+		"allowtransfer"		"true"
+		"forcedrop"		"true"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"783076"
+		"mode"			"0"
+		"maxuses"		"0"
+		"cooldown"		"0"
+		"maxamount"		"1"
+	}
+	"10"
+	{
+		"name"			"Unlimited Nades"
+		"shortname"		"Unlimited Nades"
+		"color"			"{grey}"
+		"buttonclass"		""
+		"filtername"		"he_user"
+		"hasfiltername"		"true"
+		"blockpickup"		"false"
+		"allowtransfer"		"false"
+		"forcedrop"		"false"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"679750"
+		"mode"			"0"
+		"maxuses"		"0"
+		"cooldown"		"0"
+		"maxamount"		"1"
+	}
+	"11"
+	{
+		"name"			"ZM Kart"
+		"shortname"		"ZM Kart"
+		"color"			"{default}"
+		"buttonclass"		"func_button"
+		"filtername"		"kart_user"
+		"hasfiltername"		"true"
+		"blockpickup"		"false"
+		"allowtransfer"		"false"
+		"forcedrop"		"false"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"456402"
+		"mode"			"2"
+		"maxuses"		"0"
+		"cooldown"		"35"
+		"maxamount"		"10"
+		"trigger"		"1701507"
+	}
+	"12"
+	{
+		"name"			"Bicycle"
+		"shortname"		"Bicycle"
+		"color"			"{grey}"
+		"buttonclass"		"func_button"
+		"filtername"		"bicycle_user"
+		"hasfiltername"		"true"
+		"blockpickup"		"false"
+		"allowtransfer"		"false"
+		"forcedrop"		"false"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"414940"
+		"mode"			"2"
+		"maxuses"		"0"
+		"cooldown"		"35"
+		"maxamount"		"10"
+		"trigger"		"1701580"
+	}
+	"13"
+	{
+		"name"			"Trailer"
+		"shortname"		"Trailer"
+		"color"			"{grey}"
+		"buttonclass"		"func_button"
+		"filtername"		"trailer_user"
+		"hasfiltername"		"true"
+		"blockpickup"		"false"
+		"allowtransfer"		"false"
+		"forcedrop"		"false"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"426610"
+		"mode"			"2"
+		"maxuses"		"0"
+		"cooldown"		"35"
+		"maxamount"		"10"
+		"trigger"		"1701625"
+	}
+	"14"
+	{
+		"name"			"Car"
+		"shortname"		"Car"
+		"color"			"{default}"
+		"buttonclass"		"func_button"
+		"filtername"		"car_user"
+		"hasfiltername"		"true"
+		"blockpickup"		"false"
+		"allowtransfer"		"false"
+		"forcedrop"		"false"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"443545"
+		"mode"			"2"
+		"maxuses"		"0"
+		"cooldown"		"35"
+		"maxamount"		"10"
+		"trigger"		"1701665"
+	}
+	"15"
+	{
+		"name"			"Tire"
+		"shortname"		"Tire"
+		"color"			"{default}"
+		"buttonclass"		"func_button"
+		"filtername"		"tire_user"
+		"hasfiltername"		"true"
+		"blockpickup"		"false"
+		"allowtransfer"		"false"
+		"forcedrop"		"false"
+		"chat"			"true"
+		"hud"			"true"
+		"hammerid"		"534943"
+		"mode"			"2"
+		"maxuses"		"0"
+		"cooldown"		"35"
+		"maxamount"		"10"
+		"trigger"		"1701545"
+	}
+}
```

Comment:
You need to append _override to the file name when the map already has an entWatch packed.

Reply:
If i just put .cfg without _override, will it conflict the entWatch in the map?

Reply:
Yes, it won't load your config at all because files packed in the BSP take priority.

Reply:
I see, thanks.